lib/properties/density.dart

```diff
@@ -8,8 +8,23 @@ import 'package:units_converter/utils/utils.dart';
 enum DENSITY {
   gramsPerLiter(MASS.grams, VOLUME.liters),
   gramsPerCubicCentimeter(MASS.grams, VOLUME.cubicCentimeters),
+  gramsPerMilliLiter(MASS.grams, VOLUME.milliliters),
+  gramsPerDeciLiter(MASS.grams, VOLUME.deciliters),
   kilogramsPerLiter(MASS.kilograms, VOLUME.liters),
-  kilogramsPerCubicMeter(MASS.kilograms, VOLUME.cubicMeters);
+  kilogramsPerCubicMeter(MASS.kilograms, VOLUME.cubicMeters),
+  milliGramsPerLiter(MASS.milligrams, VOLUME.liters),
+  milliGramsPerDeciLiter(MASS.milligrams, VOLUME.deciliters),
+  milliGramsPerMilliLiter(MASS.milligrams, VOLUME.milliliters),
+  milliGramsPerCubicMeter(MASS.milligrams, VOLUME.cubicMeters),
+  milliGramsPerCubicCentimeter(MASS.milligrams, VOLUME.cubicCentimeters),
+  microGramsPerLiter(MASS.micrograms, VOLUME.liters),
+  microGramsPerDeciLiter(MASS.micrograms, VOLUME.deciliters),
+  microGramsPerMilliLiter(MASS.micrograms, VOLUME.milliliters),
+  nanoGramsPerLiter(MASS.nanograms, VOLUME.liters),
+  nanoGramsPerMilliLiter(MASS.nanograms, VOLUME.milliliters),
+  picoGramsPerLiter(MASS.picograms, VOLUME.liters),
+  picoGramsPerMilliLiter(MASS.picograms, VOLUME.milliliters),
```

Comment:
Make these names more coherent. If a unit is written in muliple parts write them with camelCase
e.g. "milligrams per liter" -> milligramsPerLiter